apps/services/auth-public-api/src/app/modules/delegations/actorDelegations.controller.spec.ts

```diff
@@ -356,6 +356,28 @@ describe('ActorDelegationsController', () => {
           expect(res.status).toEqual(200)
           expect(res.body).toHaveLength(0)
         })
+
+        it('should return a legal guardian delegation since the type is included in the delegationTypes filter', async () => {
+          // Act
+          const res = await server.get(
+            `${path}${query}&delegationTypes=${DelegationType.Custom}&delegationTypes=${DelegationType.LegalGuardian}`,
+          )
+
+          // Assert
+          expect(res.status).toEqual(200)
+          expect(res.body).toHaveLength(1)
```

Comment:
I think we should add assert for the type of the delegation to make sure we get the LegalGuardian delegation.
This test might pass later on in the future with wrong type of delegation.